tests/setup-snapshot.yml

```diff
@@ -1,4 +1,4 @@
-- hosts: all
+- hosts: all,!ad
```

Comment:
Another "out of curiosity" :), how does this `hosts` setting work in the baseos ci test env?

Reply:
> Another "out of curiosity" :), how does this `hosts` setting work in the baseos ci test env?

I think this should be fine - it will run the tests on all hosts except those in the `ad` group, and that group won't exist in the baseos ci environment.  I believe this is being added so that you can run the test manually in a lab environment where you have both linux and ad machines in the same playbook.

Reply:
I will be running the integration tests in idm-ci and I will supply proper inventory. I want to make sure that if this inventory is by some mistake used by these tests it will not try to run in on the AD.